backend/config/dbConn.ts

```diff
@@ -1,3 +1,6 @@
+
+// conectare la baza de date mongoDb 
```

Comment:
Sterge comentariul asta te rog.

Nu mai adauga comentarii in cod (sau cel putin nu da push in remote). Poti fi foarte explicit sau intuitiv din numele variabilelor.